### DISCUSSION TITLE
Question about iCN900 from BiGG

### DISCUSSION THREAD

**Discussion by amarsh26** [opencobra/cobrapy#1385] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

I am receiving zero as the biomass from iCN900 running with default parameters. 
I would like to know if iCN900 from BiGG should be producing zero biomass, and if it should be, why this is the case. 
Thanks

### Code sample

Code run:

```iCN900_model=read_sbml_model('iCN900.xml')
iCN900_model.slim_optimize()
```

### Environment

<details>

</details>

### Anything else?

http://bigg.ucsd.edu/models/iCN900

**Comment by cdiener** [comment]
What medium did you apply? Without applying any medium that would be expected. You can also use the [minimal_medium](https://cobrapy.readthedocs.io/en/latest/media.html) function to check that there exists a medium that allows growth.

**Comment by amarsh26** [comment]
Thank you for your fast response. It seemed to me that it was using 
![image](https://github.com/opencobra/cobrapy/assets/67613741/a8e70c00-9fb0-4aee-b840-aa13969fe23b)
as media (this appears to be the media that "loads" with the model or is built upon loading the model). This method has worked for other BiGG models such as iML1515 and iSB619. Thanks again for your help!
